src/Auth0.ManagementApi/Clients/IGuardianClient.cs

```diff
@@ -1,7 +1,8 @@
-﻿using System.Collections.Generic;
+﻿using Auth0.ManagementApi.Models;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Auth0.ManagementApi.Models
```

Comment:
Is this public? if so, isn't it a breaking change?

Reply:
It would be however IGuardianClient and GuardianClient were never officially added to the main class that people use to access this functionality.  People would have had to have subclassed ours and added it in themselves.  If they went through that then they'll get a compile error but I doubt anyone ever did.

Reply:
I'd argue that. If the class was already pushed and scoped as public, people could have used it as you mention. e.g. The user that reported this initially. For them this is going to be a breaking change which, if we stick to semver, we must release a new major. If you can avoid this change that would be awesome. I'll leave the call to you.

Reply:
Let's bump the major then.